### ISSUE TITLE
Getting prompted for Password without Username

### ISSUE THREAD

**Issue by paolo-tanium** [cliffano/nestor#40]
When I enter the following command

```bash
JENKINS_URL=https://host nestor --interactive build build-name
```

I get prompted for my password (but not my username):

```bash
prompt: Password:
```

Furthermore, when I enter my password, I am given the following error:

```bash
  Authentication failed - incorrect username and/or password in Jenkins URL
```

**Comment by paolo-tanium** [comment]
![image](https://cloud.githubusercontent.com/assets/21039701/18697650/334cf6fe-7f91-11e6-9ea2-4167d4a40fbf.png)

**Comment by cliffano** [comment]
@paolo-tanium What Jenkins version are you using? Does it have CSRF enabled?

One possible cause to this problem is that Jenkins will return 403 with a message about invalid breadcrumb when CSRF is enabled. I'll need to add support for retrieving crumb value, and then hook it up to build job.

**Comment by paolo-tanium** [comment]
Version 2.19.1. Unfortunately I don't have admin access so I don't know if it is CSRF enabled. I'll try asking our admin tomorrow.

**Comment by cliffano** [comment]
@paolo-tanium Thanks that information would be helpful.

Another question, do you get that error only when you kick of a build? What about when you run `dashboard` command with interactive flag?

**Comment by cliffano** [comment]
@paolo-tanium I just released Nestor v1.0.1 , could you please try it?

New installations of Jenkins 2.x onward has CSRF protection enabled by default. This new version of Nestor always adds Jenkins crumb request header to solve the 403 invalid crumb problem.

**Comment by cliffano** [comment]
@paolo-tanium Closing this issue for now. Please try Nestor v1.0.1 or newer for your issue.
Please feel free to re-open if you're still experiencing the problem.
